graphitron-codegen-parent/graphitron-java-codegen/src/test/resources/resolvers/datafetchers/fetch/strategynode/oneImplementation/expected/QueryGeneratedDataFetcher.java

```diff
@@ -24,7 +24,7 @@ public static DataFetcher<CompletableFuture<Node>> node(NodeIdStrategy nodeIdStr
             var _fetcher = new DataFetcherHelper(env);
 
             switch (_typeId) {
-                case "CUSTOMER": return _fetcher.loadInterface(_loaderName, id, (ctx, ids, selectionSet) -> CustomerDBQueries.customerForNode(ctx, nodeIdStrategy, ids, selectionSet));
+                case "CustomerType": return _fetcher.loadInterface(_loaderName, id, (ctx, ids, selectionSet) -> CustomerDBQueries.customerForNode(ctx, nodeIdStrategy, ids, selectionSet));
```

Comment:
Altså sånn at dette blir en helt basic test igjen og vi får dette:
```suggestion
                case "Customer": return _fetcher.loadInterface(_loaderName, id, (ctx, ids, selectionSet) -> CustomerDBQueries.customerForNode(ctx, nodeIdStrategy, ids, selectionSet));
```